ui/src/image_annotator_polygon.ts

```diff
@@ -1,12 +1,13 @@
 import { F, S, U } from './core'
 import { DrawnPoint, DrawnShape, ImageAnnotatorPoint, ImageAnnotatorRect } from "./image_annotator"
-import { ARC_RADIUS } from "./image_annotator_rect"
+import { ARC_RADIUS_DEFAULT } from "./image_annotator_rect"
 
 export class PolygonAnnotator {
   private currPolygonPoints: ImageAnnotatorPoint[] = []
   private boundaryRect: ImageAnnotatorRect | null = null
   private draggedPoint: DrawnPoint | null = null
   private draggedShape: DrawnShape | null = null
+  private getArcRadius = () => this.ctx ? ARC_RADIUS_DEFAULT / this.ctx.getTransform().a : ARC_RADIUS_DEFAULT
```

Comment:
Can be updated in the constructor when `ctx` is set. That way arcRadius can be computed just once and thus be made a constant instead.

Reply:
The arcRadius is dependant of the currently applied transform (zoom) but constructor is called only once on image load. Therefore it cannot be constant.